README.md

```diff
@@ -221,6 +221,7 @@ It accepts the following named flags:
 
 It gets the following named parameters that can have different values in front of them:
 
+- `PREFIX`: the optional prefix that is used to define `${PREFIX}_project_options` and `${PREFIX}_project_warnings` targets when the function is used in a multi-project fashion.
```

Comment:
**IMHO**: For may reasons, this should **NOT** optional -> **it must be mandatory!**

1. konsider: project A is installed (with exported `project_options`) and later, project B, which use also  `project_options`) depends from project A. It use find_package() and import  `project_options`
2. Or Project D depends from both: A and C, but does not use  `project_options`. But A and C use independent.

One solution for this future desaster is to never export  `project_options` and  `project_warnings`!
They must be used `PRVATE `only and never exported.

Reply:
**It is not a good idea to export static-analyser settings or gcov options!**

Reply:
The default example only enables the static analyzers when the `FEATURE_TESTS ` is enabled

https://github.com/aminya/project_options#usage

Reply:
OK, I allways disable install in developer mode!

Reply:
Do not turn off the installation. Just put the test and analyzer under an option. This is very common even without this library

Reply:
I don't know which package manager you use, but the install rules are required for vcpkg and conan to work without any issues.